pokeapp/src/components/DigitalCardBinder.js

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import PokemonCard from "./PokemonCard";
 
-
-const DigitalCardBinder = ({ allPokemonFiltered, setSelectedPokemon }) => {
-
-
-  const cardPrinter = () => {
+const DigitalCardBinder = ({
+  allPokemonFiltered,
+  setSelectedPokemon,
+  myPokeList,
+  setMyPokeList,
+  isPokeList,
+  setAllPokemonFiltered,
+}) => {
+  const pokedexCardPrinter = () => {
     return allPokemonFiltered.map((pokemon) => {
       return (
         <PokemonCard
```

Comment:
You could also reduce to `const pokedexCardPrinter = () => allPokemonFilters.map`... since you are returning one thing, and it makes you look sexy 😘

Reply:
Thanks for the tip! Will do.

Reply:
Good looking out again my guy. eagle eye!